pkg/io/tfio/acl.go

```diff
@@ -88,13 +90,21 @@ func aclCollection(t *ir.ACLCollection, vpc string) *tf.ConfigFile {
 	for _, subnet := range sortedACLs {
 		comment := ""
 		vpcName := ir.VpcFromScopedResource(subnet)
-		if len(acls) > 1 {
-			comment = fmt.Sprintf("\n# %v [%v]", subnet, t.ACLs[vpcName][subnet].Internal[0].Target())
+		acl := t.ACLs[vpcName][subnet]
+		if len(sortedACLs) > 1 { // not a single nacl
+			comment = fmt.Sprintf("\n# %v [%v]", subnet, subnetCidr(acl))
 		}
-		acls[i] = singleACL(t.ACLs[vpcName][subnet], comment)
+		acls[i] = singleACL(acl, comment)
 		i += 1
 	}
 	return &tf.ConfigFile{
 		Resources: acls,
 	}
 }
+
+func subnetCidr(acl *ir.ACL) *ipblock.IPBlock {
+	if len(acl.Internal) > 0 {
+		return acl.Internal[0].Target()
+	}
+	return acl.External[0].Target()
+}
```

Comment:
Don't we have a better way to extract the CIDR of a subnet?

Reply:
not from `ir.ACL` struct

Reply:
does this information exist in `ir` at all?

Reply:
yes, in `ir.ConfigDefs`. [link](https://github.com/np-guard/vpc-network-config-synthesis/blob/main/pkg/ir/spec.go#L65).

Reply:
ok. Let's hope it won't bite us in the future.